misc/stub.dts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
as this file is being used (and printed) as being a board dts file, should we then place with the boards folder instead, like here: `boards/board-stub.dts`.

https://github.com/zephyrproject-rtos/zephyr/tree/main/boards

Reply:
That would result in following message during CMake configure:
```
"-- Found BOARD.dts: <zephyr-base>/boards/board-stub.dts
```

Reply:
done